content/1-introduction-3-problems.md

```diff
@@ -0,0 +1,129 @@
+## Open problems {#sec:intro-problems}
```

Comment:
TODO
- [ ] look for examples of each approach, and blends, in acerbi_if_2016 and sterelny_cultural_2017, to make these questions concrete